Editor/Module Management/ModuleInstaller.cs

```diff
@@ -127,9 +120,12 @@ public static void AddModuleRequest(string identifier)
         private static ModuleInfo[] GetMissingModuleNames()
         {
             PackageInfo[] installed = GetPackageList();
-            IEnumerable<ModuleInfo> missing = ModuleList.Modules.Where(m => installed.All(i => m.name != i.name));
-
-            return missing.ToArray();
+            var missingModules = ModuleList.Modules.Where(m => installed.All(i => m.name != i.name)).ToList();
+            if (ProjectPrefs.GetBool(ProjectPrefs.FIRST_TIME_SETUP_DONE))
```

Comment:
@rYuuk Any ideas on a better way to do this with a hotfix? 
Otherwise, I am somewhat tempted to just remove WebView from the installer list altogether as it would be the simplest approach that wouldn't cause errors other than lack of WebView by default :D